src/pbg3/Pbg3Archive.cpp

```diff
@@ -76,15 +76,16 @@ i32 Pbg3Archive::Release()
 
 i32 Pbg3Archive::FindEntry(char *path)
 {
-    if (this->numOfEntries == 0)
-    {
-        return -1;
-    }
+    // Why was this here? It's certainly not present in the assembly.
```

Comment:
The PBG3 code wasn't a very faithful reimpl - I was more concerned in having something that _worked_ than something that matched, so I could move on to the actual game logic faster.

So in that context, this might have been put here as a workaround or something while working on it. Anyways, you can delete these lines - they're indeed not from the original ^^.